src/screens/about/components/showcase-app.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const ShowcaseApp = (props) => (
```

Comment:
can you convert this to a React.Component component please? just to keep things consistent